app/dts/behaviors/studio_unlock.dtsi

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
```

Comment:
```suggestion
 * Copyright (c) 2024 The ZMK Contributors
```

There are a few other instances of this, e.g. in the binding yaml. Didn't note all not to pollute the review too much

Reply:
Fixed them all up, and added a few missing ones. Thanks.